database/README.md

```diff
@@ -309,6 +326,56 @@ Indexes:
 - PRIMARY KEY: btree(`id`)
 - quotes\_token\_expiration: btree (`sell_token`, `buy_token`, `expiration_timestamp` DESC)
 
+### proposed\_solutions
+
+All solutions reported by solvers, that were part of a solver competition.
+
+ Column        | Type      | Nullable | Details
+---------------|-----------|----------|--------
+ auction\_id   | bigint    | not null | auction for which the solution was proposed 
+ uid           | bigint    | not null | unique id of the proposed solution within a single auction
+ id            | bytea     | not null | id of the proposed solution as reported by the solver
+ solver        | bytea     | not null | solver submission address
+ is\_winner    | boolean   | not null | specifies if a solution was chosen as a winner of a competition
```

Comment:
might be worthwhile to mention that there may be multiple winners in an auction and this mainly means the solver was requested to execute the solution.